### PR TITLE
#3 create html for homepage

### PR DESCRIPTION
**Pull request by stephanemalho**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- add this font on css font-family: 'Shrikhand', cursive,'Roboto', sans-serif; -->
+    <!--HEADER-->
+    <!--HEADER-->
+    <header>
+      <h1><a href="#goToFoot" id="goToTitle">ohmyfood</a></h1>
```
Comment (Tutt-J) [inline index.html:33]: What is this href ?? id is not needed here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      <!--TO GO TO FOOTER INFO-->
+      <div class="datalist-holder">
+        <i class="fa-map-marker-alt fas"></i>
+        <select size="1" name="district">
```
Comment (Tutt-J) [inline index.html:37]: Please be simple no select just a p with icon

File: index.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    />
+  </head>
+  <body>
+    <!-- add this font on css font-family: 'Shrikhand', cursive,'Roboto', sans-serif; -->
+    <!--HEADER-->
+    <!--HEADER-->
+    <header>
+      <h1><a href="#goToFoot" id="goToTitle">ohmyfood</a></h1>
+      <!--TO GO TO FOOTER INFO-->
+      <div class="datalist-holder">
```
Comment (Tutt-J) [inline index.html:35]: remove this class and use section with id

File: index.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+          <option>Paris, Faidherbes</option>
+          <option>Paris, Bastille</option>
+        </select>
+      </div>
+    </header>
+    <!--AND HEADER-->
+    <!--OPEN MAIN-->
+    <!--OPEN MAIN-->
+    <main>
+      <section>
```
Comment (Tutt-J) [inline index.html:51]: add id

File: index.html [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+        <h2>Réservez le menu qui vous convient</h2>
+        <p>Découvrez des restaurants d'exeption, séléctionnés par nos soins.</p>
+        <a href="#">Explorer nos restaurants</a>
+      </section>
+      <section>
```
Comment (Tutt-J) [inline index.html:56]: add id

File: index.html [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+      </div>
+    </header>
+    <!--AND HEADER-->
+    <!--OPEN MAIN-->
+    <!--OPEN MAIN-->
+    <main>
+      <section>
+        <h2>Réservez le menu qui vous convient</h2>
+        <p>Découvrez des restaurants d'exeption, séléctionnés par nos soins.</p>
+        <a href="#">Explorer nos restaurants</a>
```
Comment (Tutt-J) [inline index.html:54]: href must be anchor

File: index.html [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+          </li>
+          <li>
+            <a href="#">
+              <i class="fas fa-store"></i>
+              <p>Degustez au restaurant</p>
+            </a>
+          </li>
+        </ol>
+      </section>
+      <section>
```
Comment (Tutt-J) [inline index.html:79]: add id

File: index.html [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+        <h3>Restaurants</h3>
+        <article>
+          <figure>
+            <a href="#">
+              <div>
```
Comment (Tutt-J) [inline index.html:84]: remove div no util

File: index.html [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
+              </a>
+              <i class="far fa-heart"></i>
+            </figcaption>
+          </figure>
+        </article>
+      </section>
+    </main>
+    <footer>
+      <div>
+        <h5><a href="#goToTitle" id="goToFoot">ohmyfood</a></h5>
```
Comment (Tutt-J) [inline index.html:163]: what is this href ??

File: index.html [hunk 10]
```diff
@@ -0,0 +1,186 @@
...
+        <!--TO RETURN TO HEADER-->
+        <nav>
```
Comment (Tutt-J) [inline index.html:165]: use div

File: index.html [hunk 11]
```diff
@@ -0,0 +1,186 @@
...
+          <ul>
+            <li>
+              <a href="#"
+                ><i class="fas fa-utensils"></i>&nbsp;&nbsp;Proposez un
```
Comment (Tutt-J) [inline index.html:169]: remove $nbsp

File: index.html [hunk 12]
```diff
@@ -0,0 +1,186 @@
...
+                restaurant</a
+              >
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fas fa-hands-helping"></i>&nbsp;&nbsp;Devenir
```
Comment (Tutt-J) [inline index.html:175]: remove &nbsp

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (stephanemalho) [reply index.html:33]: c'est pour faire une ancre vers le footer et du footer vers le header

File: index.html [hunk 13]
```diff
@@ -0,0 +1,170 @@
...
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- add this font on css font-family: 'Shrikhand', cursive,'Roboto', sans-serif; -->
+    <!--HEADER-->
+    <!--HEADER-->
+    <header> 
+      <section id="cityChoise">
+        <h1><a>ohmyfood</a></h1>
```
Comment (Tutt-J) [inline index.html:34]: href must be to index.html

File: index.html [hunk 14]
```diff
@@ -0,0 +1,170 @@
...
+              </a>
+              <i class="far fa-heart"></i>
+            </figcaption>
+          </figure>
+        </article>
+      </section>
+    </main>
+    <footer>
+      <nav>
+        <h5><a>ohmyfood</a></h5>
```
Comment (Tutt-J) [inline index.html:148]: href mut be to index.html

File: index.html [hunk 15]
```diff
@@ -0,0 +1,170 @@
...
+      </section>
+    </header>
+    <!--AND HEADER-->
+    <!--OPEN MAIN-->
+    <!--OPEN MAIN-->
+    <main>
+      <section id="menu">
+        <h2>Réservez le menu qui vous convient</h2>
+        <p>Découvrez des restaurants d'exeption, séléctionnés par nos soins.</p>
+        <a href="#restaurant">Explorer nos restaurants</a>
```
Comment (Tutt-J) [inline index.html:45]: #restaurants with s

File: index.html [hunk 16]
```diff
@@ -0,0 +1,170 @@
...
+          </li>
+          <li>
+            <a href="#">
+              <i class="fas fa-store"></i>
+              <p>Degustez au restaurant</p>
+            </a>
+          </li>
+        </ol>
+      </section>
+      <section id="restaurant">
```
Comment (Tutt-J) [inline index.html:70]: restaurant with s

**Review by Tutt-J — CHANGES_REQUESTED** [review]

**Review by Tutt-J — APPROVED** [review]

**Pull request merged**